reviewrot/basereview.py

```diff
@@ -1,8 +1,41 @@
 from collections import OrderedDict
+import datetime
+from dateutil.relativedelta import relativedelta
 
 
 class BaseService(object):
 
+    def format_reviews(self, username, title, url, comments, created_at,
+                       state_, value, duration):
+        """
+        find the relative time difference between now and
+        review request filed to retrieve relative information
+        """
+        rel_diff = relativedelta(datetime.datetime.now(),
+                                 created_at)
+        """
+        find the absolute time difference between now and
+        review request filed to retrieve absolute information
+        """
+        abs_diff = datetime.datetime.now() - created_at
+        if (state_ is not None and value is not None and
```

Comment:
I don't think this belongs in this method since it doesn't have anything to do with formatting a review and causes two different types of return values which is never a good idea. Perhaps you can have a method with checks if a review should be included.